edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -0,0 +1,158 @@
+edx = edx || {};
+
+(function(Backbone, $) {
+    'use strict';
+
+    var examStatusReadableFormat, notStartedText, startedText, submittedText;
+
+    edx.courseware = edx.courseware || {};
+    edx.courseware.proctored_exam = edx.courseware.proctored_exam || {};
+
+    notStartedText = {
+        status: gettext('Not Started'),
+        message: gettext('You have not started your onboarding exam.')
+    };
+    startedText = {
+        status: gettext('Started'),
+        message: gettext('You have started your onboarding exam.')
+    };
+    submittedText = {
+        status: gettext('Submitted'),
+        message: gettext('You have submitted your onboarding exam.')
+    };
+
+    examStatusReadableFormat = {
+        created: notStartedText,
+        download_software_clicked: notStartedText,
+        ready_to_start: notStartedText,
+        started: startedText,
+        ready_to_submit: startedText,
+        second_review_required: submittedText,
+        submitted: submittedText,
+        verified: {
+            status: gettext('Verified'),
+            message: gettext('You can now take proctored exams in this course.')
+        },
+        rejected: {
+            status: gettext('Rejected'),
+            message: gettext('Your onboarding exam has been rejected. Please retry onboarding.')
+        },
+        error: {
+            status: gettext('Error'),
+            message: gettext('An error has occurred during your onboarding exam. Please retry onboarding.')
+        }
+    };
+
+    edx.courseware.proctored_exam.ProctoredExamInfo = Backbone.View.extend({
+        initialize: function() {
+            this.course_id = this.$el.data('course-id');
+            this.model.url = this.model.url + '?course_id=' + encodeURIComponent(this.course_id);
+            this.template_url = '/static/proctoring/templates/proctored-exam-info.underscore';
+            this.status = '';
+
+            this.loadTemplateData();
+        },
+
+        updateCss: function() {
```

Comment:
#NIT there is no good way to update a css label instead? We need to update different attribute on CSS 1 by 1?

Reply:
Looking through the other Backbone files in the repo it looks like this is how custom CSS is added to the templates. I would feel comfortable leaving this so it follows the patterns of the other files